bears/csv/CsvLintBear.py

```diff
@@ -0,0 +1,10 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+@linter(executable='csvlint',
+        output_format='regex',
+        output_regex=r'INVALID',
```

Comment:
because the output the tool gives is like:
``..............................................................( * 50 lines) VALID``